NOTICE

```diff
@@ -0,0 +1,4 @@
+Datadog kafka-connect-datadog-logs
```

Comment:
`kafka-connect-datadog-logs` was renamed to `datadog-kafka-connect-logs`. There are still few occurences of `kafka-connect-datadog-logs`

Reply:
Let me check, I thought I caught them all